### DISCUSSION TITLE
Error when attempting to delegate

### DISCUSSION THREAD

**Discussion by pakalatus** [ping-pub/explorer#543] [Q&A]
My community is trying to delegate to a validator (pakalatus-node on Artela testnet) via Cosmotracker, but they keep getting the following error: "Error: unable to resolve type URL /ethermint.crypto.v1.ethsecp256k1.PubKey: tx parse error."".

Could you please advice the resolution?

**Comment by kenanilgun** [comment]
Yes, also I have same problem on my fork, I got reference from chains.d.ts and tried use many configuration but not change anything.

**Comment by liangping** [comment]
for EVM chain, they might need some customization for key type. There's no common solution if they don't support `/ethermint.crypto.v1.ethsecp256k1.PubKey`
